include/aws/http/http.h

```diff
@@ -50,6 +50,7 @@ enum aws_http_errors {
     AWS_ERROR_HTTP_PROXY_STRATEGY_TOKEN_RETRIEVAL_FAILURE,
     AWS_ERROR_HTTP_PROXY_CONNECT_FAILED_RETRYABLE,
     AWS_ERROR_HTTP_PROTOCOL_SWITCH_FAILURE,
+    AWS_ERROR_HTTP_STREAM_EXCEED_MAX_CONCURRENT,
```

Comment:
trivial: I'd suggest AWS_ERROR_HTTP_MAX_CONCURRENT_STREAMS_EXCEEDED

it reads a little bit more naturally, and the official name of the setting is MAX_CONCURRENT_STREAMS